parts/linux/cloud-init/artifacts/components.json

```diff
@@ -352,8 +332,7 @@
       "multiArchVersionsV2": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=aks/ip-masq-agent-v2",
-          "latestVersion": "v0.1.15",
-          "previousLatestVersion": "v0.1.14"
+          "latestVersion": "v0.1.15"
```

Comment:
this one makes me nervous since ip-masq-agent is running on almost every AKS cluster. Could we keep it?

Reply:
sure, this one's relatively small actually